feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -270,15 +270,28 @@ func (tc *testCase) verifyInstall(ctx context.Context, t *testing.T) {
 	}
 
 	if tc.dualSup {
-		if got, want := r.GetVerifyStandby().GetVerifyResponse().GetActivationFailMessage(), ""; got != want {
-			t.Errorf("OS.Verify Standby ActivationFailMessage: got %q, want %q", got, want)
-		}
+		var counter int
+		const maxRetries = 15
+		for i := 1; i < maxRetries; i++ {
+			r, err := tc.osc.Verify(ctx, &ospb.VerifyRequest{})
+			if err != nil {
+				t.Fatalf("OS.Verify request failed: %s", err)
+			}
+			if counter == maxRetries {
+				t.Fatalf("OS.Verify Standby could not obtain version status in 15 minutes")
+			}
+			counter++
 
-		if got, want := r.GetVerifyStandby().GetVerifyResponse().GetVersion(), *osVersion; got != want {
-			t.Errorf("OS.Verify Standby Version: got %q, want %q", got, want)
+			if got, want := r.GetVerifyStandby().GetVerifyResponse().GetActivationFailMessage(), ""; got != want {
+				t.Errorf("OS.Verify Standby ActivationFailMessage: got %q, want %q", got, want)
+			}
+
+			if got, want := r.GetVerifyStandby().GetVerifyResponse().GetVersion(), *osVersion; got != want {
+				t.Logf("Attempt %d/15 - OS.Verify Standby Version: got %q, want %q", counter, got, want)
+				time.Sleep(1 * time.Minute)
```

Comment:
@liulk is it `time.Sleep() + for loop` the recommended way in FP test to achieve the timeout logic? Or, should we use something else e.g. compare the timeStamp? Thanks.